pkg/elevate/elevate.go

```diff
@@ -55,6 +55,19 @@ func RunElevate(argv []string) error {
 		return err
 	}
 
+	// As WriteKubeconfigToFile(utils.CreateTempKubeConfig) is overriding KUBECONFIG,
+	// we need to store its definition in order to redefine it to its original (value or unset) when we do not need it anymore
+	oldKubeconfigPath, oldKubeconfigDefined := os.LookupEnv("KUBECONFIG")
```

Comment:
I guess, this may align with `login multi-cluster` subcommand and `session` commands

Reply:
No this is out of context of multi-cluster/session as it is only for elevate